src/Symfony/Component/CssSelector/XPath/XPathExpr.php

```diff
@@ -78,15 +64,12 @@ public function addCondition($condition)
     /**
      * @return string
      */
-    public function getCondition()
+    public function getCondition(): string
     {
         return $this->condition;
     }
 
-    /**
-     * @return $this
```

Comment:
the class is internal and the return type ensures this allows for chaining calls, if no objection I'd keep removing them